sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala

```diff
@@ -207,6 +207,26 @@ class SQLQuerySuite extends QueryTest {
     }
   }
 
+  test("CTE feature") {
```

Comment:
You might consider putting these tests in HiveQuerySuite, as then the answers will be autogenerated.  Also can you add tests for the `FROM src SELECT *` case?